packer/build.pkr.hcl

```diff
@@ -8,7 +8,7 @@ build {
   }
 
   provisioner "ansible-local" {
-    playbook_dir = "../ansible"
-    playbook_file = "../ansible/playbook.yml"
+    playbook_dir = "../ansible/"
+    playbook_file = "../ansible/add_prereqs.yml"
```

Comment:
is it `prerequisites`?

Reply:
Eh.. in my head it is `prereqs for box to run`, but that might be a stretch for sure. Thoughts? 

I'm all ears!

Reply:
`prestart_playbook.yml` ?

Reply:
Makes it confusing with the `prestart` `poststart` notion we got going on. This is before the `prestart`. We could change the name of `prestart` `poststart` to `pre_hashi` and `post_hashi`maybe?

Reply:
I double-checked, `add_prereqs.yml` was main `playbook.yml` before. 
What about renaming it to `build.yml` or `entry_build.yml`?

Reply:
Yeah, that's right, it was the main playbook. I like something along the line of entry, but it doesn't build much, does it? A lot of copying and moving files around. What about `bootstrap`. It is essentially a bootstrap of the environment the box is supposed to supply.

Reply:
`bootstrap` makes sense. @fredrikhgrelland WDYT?

ps. naming the most difficult part in IT

Reply:
renamed it to `bootstrap`.